### PR TITLE
Initial content

### PR DESCRIPTION
**Pull request by andygrunwald**
Chapters:

* Where i can find you?
* Where can i get more information or provide feedback?
* Where is the original place of this content?

Thinks that might we want to write down:

* Formats
* How we find speakers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
- Everything you need to know about how we run the Web Engineering DUS Meetup.
+# WebEngDUS
+
+Everything you need to know about how we run the *Web Engineering DUS Meetup*.
+
+## Where i can find you?
+
+The *Web Engineering Düsseldorf* is available at several places:
+
+* [Web Engineering Düsseldorf at Meetup](https://www.meetup.com/Web-Engineering-Duesseldorf/)
```
Comment (dsiebel) [inline README.md]: on Meetup, on Tiwtter, on GitHub

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+* [@WebEngDUS at twitter](https://twitter.com/WebEngDUS)
+* [WebEngDUS at GitHub](https://github.com/WebEngDUS)
+
+## Where can i get more information or provide feedback?
+
+If you want to get more information and you have a specific question, feel free to [open an issue at our GitHub repository](https://github.com/WebEngDUS/WebEngDUS/issues/new).
```
Comment (dsiebel) [inline README.md]: "**or** you have a specific question"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,22 @@
...
+We will be happy to answer it.
+
+When you want to provide us feedback to this content, to the way we run the meetup or to the meetup itself, [open an issue at our GitHub repository](https://github.com/WebEngDUS/WebEngDUS/issues/new) is also the way to go.
```
Comment (dsiebel) [inline README.md]: "**If** you want to provide us feedback **regarding** this content, the way we run the meetup or anything else, open**ing** and issue ..."

**Review by dsiebel — CHANGES_REQUESTED** [review]
Added a couple of comments regarding typos

**Comment by andygrunwald** [comment]
@dsiebel Check again and feel free to merge

**Review by dsiebel — APPROVED** [review]

**Pull request merged**